pep8.py

```diff
@@ -2061,7 +2061,8 @@ def read_config(options, args, arglist, parser):
         # Second, parse the configuration
         for opt in config.options(pep8_section):
             if opt.replace('_', '-') not in parser.config_options:
-                print("  unknown option '%s' ignored" % opt)
+                if options.verbose:
+                    print("  unknown option '%s' ignored" % opt)
```

Comment:
Also, I saw one support issue where a user's config `verbose` settings.
https://secure.helpscout.net/search/convos/?query=pep8

We probably want to add `--quiet` to docker command
https://github.com/codeclimate/pep8/blob/jp-code-climate-support/Dockerfile#L13